lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,100 @@
+require 'rake'
+
+$organization_id = 13856
+
+def remove_teem_group_member(token, tm_diff_users, all_user_ids, group_id)
+  tm_diff_users.each do |email|
+    user_id = all_user_ids[email]
+
+    user = Teem.teem_get_user(token, user_id)
+    user_groups = user["group_ids"]
+    user_groups.delete(group_id)
+
+    Teem.teem_update_group_assign(token, user_groups, user_id)
+  end
+end
+
+def add_rm_group_member(token, rm_diff_users, all_user_ids, group_id, role)
+  rm_diff_users.each do |email|
+    user_id = all_user_ids[email]
+
+    if user_id
+      Teem.teem_update_group_assign(token, group_id, user_id)
+    else
+      role.members().each do |member|
```

Comment:
`role.members.each` is preferred over `role.members().each`. It's the exact same code, but in Ruby you can and should omit the function call parentheses when they're not needed.